app/Services/UserService.php

```diff
@@ -23,4 +26,13 @@ public function updateUser(array $userData, User $user)
 
         $user->update();
     }
+
+    public function deleteUser(array $userData, User $user)
+    {
+        $user->status = User::Inactive;
+        $user->update();
+        $pdf = Pdf::loadView('pdf.user', $user->toArray());
+        $content = $pdf->download()->getOriginalContent();
+        Storage::put('public/user.pdf', $content);
```

Comment:
move send email logic here. Instead of saving pdf to storage -> attach it to email directly.